### ISSUE TITLE
Question: Can this library calculate seawater vapor pressure?

### ISSUE THREAD

**Issue by kevinsmia1939** [TEOS-10/GSW-Python#121]
Hi,

I wonder if this or any other python library can calculate vapor pressure of seawater given temperature, pressure, and salinity?
I found a correlation of vapor pressure on this paper, https://doi.org/10.1016/j.desal.2016.02.024

**Comment by ocefpaf** [comment]
I don't believe that is in the c-library that we wrap here. Maybe that is part of the SIA library? See https://www.teos-10.org/software.htm
